python/frontend/cirrus/cirrus/lambdas/feature_hashing_helper.py

```diff
@@ -0,0 +1,31 @@
+""" Helper functions for feature hashing """
+
+import hashlib
+import json
+import struct
+
+import boto3
+
+
+def hash_data(data, columns, N):
+    """ Replace the appropriate columns for this row. """
+    col_set = set([int(i) for i in columns])
+    for idx, row in enumerate(data):
+        row_map = {}
+        for col, val in row:
+            s_c = int(col)
+            if s_c in col_set:
+                hasher = hashlib.sha256()
```

Comment:
This should not be created every time. Also we should use MurmurHash.

https://pypi.org/project/mmh3/

Reply:
I think it should be created every time if we're using hashlib, but I've switched to MurmurHash regardless